hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -158,6 +160,16 @@ const (
 // NoopReconcile is just a default mutation function that does nothing.
 var NoopReconcile controllerutil.MutateFn = func() error { return nil }
 
+var capiRelatedManagedResourcesToWatch = []client.Object{
```

Comment:
AWSEndpointService is not a capi resource and there might be others. Let's rather use platform as the driver, e.g. platformSpecificResources